bin/build_olmsted_data.py

```diff
@@ -139,22 +149,40 @@ def create_node_records(tree):
             n.parent = None
             n.length = 0.0
             n.distance = 0.0
-        records.append({'id': n.id, 'label': n.label, 'type': n.type, 'parent': n.parent, 'length': n.length, 'distance': n.distance, 'height': n.height})
+        records.append({'id': n.id, 'label': n.label, 'type': n.type, 'parent': n.parent, 'length': n.length, 'distance': n.distance, 'height': n.height, 'nt_seq': n.nt_seq, 'aa_seq': n.aa_seq, 'multiplicity': n.multiplicity, 'cluster_multiplicity': n.cluster_multiplicity})
 
     return records
 
-def parse_tree_data(s):
+def parse_tree_data(s, nt_seqs_dict, aa_seqs_dict, seqmeta_dict):
     t = PhyloTree(s, format=1)
-    return create_node_records(t)
+    return create_node_records(t, nt_seqs_dict, aa_seqs_dict, seqmeta_dict)
+
+def parse_fasta_string(fasta_string):
```

Comment:
This is cute, but should use Biopython instead.

```
from Bio import SeqIO
seqs = SeqIO.parse(filename, 'fasta')
for seqrecord in seqs:
    print "seq data", seqrecord.id, seqrecord.seq
```

Reply:
No longer using this parser as of latest commit